inception-search-core/src/main/java/de/tudarmstadt/ukp/inception/search/SearchService.java

```diff
@@ -36,5 +36,6 @@
     public Index getIndex(Project aProject);
 
     public void updateIndex(Index aIndexObject);
-
+    
+    public boolean isIndexValid(Project aProject);
```

Comment:
I think `indexIsValid` or `hasValidIndex` sounds better

Reply:
I think both your suggestions lead to a misinterpretation of the method's goal. 1) indexIsValid does not sound as a question. 2) As for hasValidIndex, the method must answer the question about the state of the index, not about its existence.